client/verta/verta/client.py

```diff
@@ -119,49 +125,31 @@ class Client(object):
 
     """
     def __init__(self, host=None, port=None, email=None, dev_key=None,
-                 max_retries=5, ignore_conn_err=False, use_git=True, debug=False, extra_auth_headers={}, _connect=True):
+                 max_retries=5, ignore_conn_err=False, use_git=True, debug=False, extra_auth_headers={}, _connect=True, jwt_token=None, jwt_token_sig=None):
```

Comment:
I think new public parameters should go before our private parameters (plus I highly doubt we're passing `_connect` positionally), though my view might be incorrect!
```suggestion
                 max_retries=5, ignore_conn_err=False, use_git=True, debug=False, extra_auth_headers={}, jwt_token=None, jwt_token_sig=None, _connect=True):
```